hs_core/tasks.py

```diff
@@ -319,7 +321,7 @@ def send_over_quota_emails():
                                   [u.email, settings.DEFAULT_SUPPORT_EMAIL],
                                   html_message=msg_str)
                     except Exception as ex:
-                        logger.debug("Failed to send quota warning email: " + ex.message)
+                        logger.debug("Failed to send quota warning email: " + str(ex))
```

Comment:
noticed a few more of these `ex.message` so I fixed them and pushed. Reject if you don't like! Hope that's ok

Reply:
Thanks for fixing this.